tox.ini

```diff
@@ -17,13 +18,15 @@ setenv =
     PYTHONHASHSEED=0
     TESTS_DIR={[testenv]unit_tests}
     non-utc-tz: TZ=EST+5
-    PYTHON=coverage run --source hotsos --parallel-mode
+    coverage: PYTHON=coverage run 
+    coverage: COV_ERASE=coverage erase
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/test-requirements.txt
 commands =
-    coverage erase
-    stestr run --serial --test-path {[testenv]unit_tests} {posargs}
+    # Wrap stestr in bash and handle the interrupt so we can clean after it
+    coverage: coverage erase
+    bash -c 'function inth()\{ {env:COV_ERASE:/bin/true}; \}; trap inth INT; stestr run --serial --test-path {[testenv]unit_tests} {posargs}'
```

Comment:
It's weird since it's already listed in `allowlist_externals`. I'll use the explicit path of the bash, i.e. "/usr/bin/bash" to see whether that removes the warning.

Reply:
Yeah, that gets rid of the warning.